core-bundle/contao/classes/BackendUser.php

```diff
@@ -132,6 +138,9 @@ public function __get($strKey)
 
 			case 'fop':
 				return \is_array($this->arrData['fop']) ? $this->arrData['fop'] : ($this->arrData['fop'] ? array($this->arrData['fop']) : false);
+
+			case 'alexf':
+				return $this->alexf;
```

Comment:
```suggestion
			case 'alexf':
				return (array) $this->arrData['alexf'] ?? array();
```
Can’t we use something like this instead of reintroducing the member variable?

Or do we need to use `StringUtil::deserialize()`?

Reply:
What is the downside of having the member variable?

Reply:
Since `$this->$k = …` was replaced with `$this->arrData[$k] = …` in #7365 your change would not work as expected becuase the data would get stored in `$this->arrData` but read from `$this->alexf`